lib/wallaroo_labs/messages/external_messages.pony

```diff
@@ -39,6 +39,14 @@ primitive _PartitionCountQueryResponse              fun apply(): U16 => 13
 primitive _SourceIdsQuery                           fun apply(): U16 => 14
 primitive _SourceIdsQueryResponse                   fun apply(): U16 => 15
 primitive _ReportStatus                             fun apply(): U16 => 16
+primitive _StateEntityQuery                         fun apply(): U16 => 17
+primitive _StateEntityQueryResponse                 fun apply(): U16 => 18
+primitive _StatelessPartitionQuery                  fun apply(): U16 => 19
```

Comment:
`_StatelessPartitionsQuery` and similar below

Reply:
see #2268